receiver/sqlserverreceiver/scraper.go

```diff
@@ -299,3 +321,252 @@ func (s *sqlServerScraperHelper) recordDatabaseStatusMetrics(ctx context.Context
 
 	return errors.Join(errs...)
 }
+
+func (s *sqlServerScraperHelper) recordDatabaseSampleQuery(ctx context.Context) (plog.Logs, error) {
+	// Constants are the column names of the database status
+	const DBName = "db_name"
```

Comment:
Could we follow the semantic convention for common attributes? https://github.com/open-telemetry/semantic-conventions/blob/main/docs/database/database-spans.md#common-attributes

For instance, `db_name` should become [`db.collection.name`](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/attributes-registry/db.md)

For those that are belong to sql server specified field. We should use the original name from sql server and add `sqlserver` prefix to prevent attribute conflict in the future.

For instance, `query_hash` should be `sqlserver.query_hash`.

Reply:
updated based on this comment and the comment you made in the other pr